app/cli/internal/action/attestation_init.go

```diff
@@ -70,7 +70,7 @@ func NewAttestationInit(cfg *AttestationInitOpts) (*AttestationInit, error) {
 }
 
 // returns the attestation ID
-func (action *AttestationInit) Run(ctx context.Context, contractRevision int, projectName, workflowName string) (string, error) {
+func (action *AttestationInit) Run(ctx context.Context, contractRevision int, projectName, workflowName, newWorkflowContractName string) (string, error) {
```

Comment:
This changes the current behaviour a little bit. Currently, contract is not mandatory.

Reply:
It is not mandatory here either. The name can be empty string